openapi.yaml

```diff
@@ -10142,3 +10187,65 @@ components:
       - account
       - name
       type: object
+    FilterEnvironmentsRequest:
+      additionalProperties: false
+      description: Request body of `/filter/environments`. Filters for deployment
+        environments.
+      example:
+        repositories:
+        - github.com/athenianco/athenian-api
+        - github.com/athenianco/athenian-webapp
+        date_to: 2022-02-23
+        account: 1
+        date_from: 2022-01-23
+      properties:
+        account:
+          description: Account ID.
+          type: integer
+        repositories:
+          allOf:
+          - description: |
+              At least one repository in the list must be deployed in `[date_from, date_to)`.
+          - $ref: '#/components/schemas/RepositorySet'
+        date_from:
+          description: Date from when to start looking for deployments.
+          format: date
+          type: string
+        date_to:
+          description: Date until which to look for deployments.
+          format: date
+          type: string
+        timezone:
+          $ref: '#/components/schemas/TimeZone'
+      required:
+      - account
+      - date_from
+      - date_to
+      type: object
+    FilteredEnvironments:
+      description: List of distinct deployment environments.
+      items: 
+        $ref: '#/components/schemas/FilteredEnvironment'
+      type: array
+    FilteredEnvironment:
+      additionalProperties: false
+      description: Details about a deployment environment.
+      properties: 
+        name:
+          description: Name of the environment.
+          type: string
+        deployments_count:
+          description: Number of deployments (successful or not) in the specified time interval.
+          minimum: 1
+          type: integer
+        last_conclusion:
```

Comment:
This is not necessary

Reply:
I followed GitHub commit statuses here. They show either a green check mark or a red cross for the latest commit in the repository.

Reply:
That's interesting, we won't use it in the MVP, but get your point, will likely be useful as an enhancement.